integrations/src/main/scala/com/quantemplate/integrations/commands/addresscleansing/AddressCleansingCmd.scala

```diff
@@ -0,0 +1,133 @@
+package com.quantemplate.integrations.commands.addresscleansing
+
+
+import java.nio.file.Path
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.scaladsl.Behaviors
+import akka.actor.typed.scaladsl.adapter.given
+import akka.actor.Scheduler
+import akka.pattern.retry
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.given
+import scala.util.{Failure, Success}
+import org.slf4j.LoggerFactory
+import org.slf4j.Logger
+import com.google.maps.model.AddressComponentType.*
+import cats.syntax.option.given
+import cats.syntax.traverse.given
+
+import com.quantemplate.integrations.common.*
+import com.quantemplate.integrations.capitaliq.CapitalIQService
+import com.quantemplate.integrations.qt.QTService
+import com.quantemplate.integrations.qt.QTModels.*
+import com.quantemplate.integrations.commands.IdentifierLoader
+import com.quantemplate.integrations.gmaps.GeocodingService
+
+class AddressCleansingCmd:
+  import AddressCleansingCmd.*
+  import AddressCleanseError.*
+
+  private given Config.GoogleMaps = Config.GoogleMaps.load()
+  private given Config.Quantemplate = Config.Quantemplate.load()
+
+  private given logger: Logger = LoggerFactory.getLogger(getClass)
+  private given system: ActorSystem[Nothing] = ActorSystem(Behaviors.empty, "api-integrations")
+  private given Scheduler = system.scheduler.toClassic
+  private given ExecutionContext = system.executionContext
+
+  private lazy val qtService = QTService(HttpService())
+  private lazy val geocodingService = GeocodingService()
+
+  def fromConfigFile(config: AddressCleansingConfigDef) =
+    import config.{ orgId } 
+    import config.source.pipeline.{ pipelineId, outputName, dataColumn, idColumn }
+    import config.target.{ dataset as targetDatasetId, onFinished }
```

Comment:
Is there any specific reason not to do these imports  at the top of the file?

Reply:
Those are imports from the function argument, I think it makes the code below more readable.